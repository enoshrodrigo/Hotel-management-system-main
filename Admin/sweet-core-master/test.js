// @flow
import { RawDelimiter } from 'sweet-spec';

let t = new RawDelimiter();
