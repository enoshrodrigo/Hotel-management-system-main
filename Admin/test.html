<!-- <!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <form id = "policeForm" action="" style="display:none">
        <input type="submit" value="Police Form">
      </form>
      <form id = "citizenForm" action="" style="display:none">
        <input type="submit" value="Citizen Form">
      </form>
      <button id= "citizenBtn" onclick="showForm('citizenForm')">citizen</button>
      <button id="police" onclick="showForm('policeForm')">police</button>


      <script>
          function showForm(){
            document.getElementById('pop').style.display ='';
          }
          </script>
</body>
</html> -->


