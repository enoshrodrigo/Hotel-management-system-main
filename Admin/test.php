<!DOCTYPE html>
<html>
<head>
  <title>Login Page</title>
  <style>
    /* Hide the login form by default */
    .login-container {
      display: none;
    }
    /* Style the login form */
    .login-form {
      position: fixed;
      top: 50%;
      left: 50%;
      transform: translate(-50%, -50%);
      background-color: #f2f2f2;
      padding: 20px;
      border-radius: 5px;
      box-sizing: border-box;
      width: 80%;
      max-width: 400px;
    }
    /* Style the form inputs */
    .login-form input[type="text"], .login-form input[type="password"] {
      margin-bottom: 20px;
      padding: 12px 20px;
      border-radius: 4px;
      border: 1px solid #ccc;
      box-sizing: border-box;
      width: 100%;
    }
    /* Style the submit button */
    .login-form input[type="submit"] {
      background-color: #4CAF50;
      color: white;
      padding: 12px 20px;
      border: none;
      border-radius: 4px;
      cursor: pointer;
      width: 100%;
    }
  </style>
</head>
<body>
  <button id="login-button">Login</button>
  <div class="login-container">
    <form class="login-form" action="<?php echo $_SERVER['PHP_SELF']; ?>" method="post">
      <label for="username">Username:</label>
      <input type="text" id="username" name="username">
      <br>
      <label for="password">Password:</label>
      <input type="password" id="password" name="password">
      <br><br>
      <input type="submit" value="Login">
    </form>
  </div>
  <script>
    // Get the login button and login container elements
    var loginButton = document.getElementById("login-button");
    var loginContainer = document.querySelector(".login-container");
    // Add an event listener to the login button
    loginButton.addEventListener("click", function() {
      // When the button is clicked, toggle the display of the login container
      loginContainer.style.display = (loginContainer.style.display === "none") ? "block" : "none";
    });
  </script>
  <?php
    if ($_SERVER['REQUEST_METHOD'] === 'POST') {
      $username = $_POST['username'];
      $password = $_POST['password'];
      
      // Check if the username and password match a user in the database
      // If a match is found, log the user in
      // If no match is found, display an error message
    }
  ?>
</body>
</html>
