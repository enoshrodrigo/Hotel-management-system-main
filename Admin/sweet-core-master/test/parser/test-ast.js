import test from 'ava';
import { getAst } from '../assertions';

test('does not include the lang directive in the AST', t => {
  t.snapshot(
    getAst(`
    'lang sweet.js';
  `),
  );
});

test('does include the use strict directive in the AST', t => {
  t.snapshot(
    getAst(`
    'use strict';
  `),
  );
});

test('includes export in AST', t => {
  t.snapshot(
    getAst(`
    export { b }
    `),
  );
});

test('includes export with renaming in AST', t => {
  t.snapshot(
    getAst(`
    export { b as c}
    `),
  );
});

test('includes export declaration in AST', t => {
  t.snapshot(
    getAst(`
    export var x = 1;
    `),
  );
});

test('includes support for async function declarations', t => {
  t.snapshot(
    getAst(`
    async function f() {}
    `),
  );
});

test('includes support for async function expressions', t => {
  t.snapshot(
    getAst(`
    let f = async function f() {}
    `),
  );
});

test('includes support for exporting async functions', t => {
  t.snapshot(
    getAst(`
    export async function f() {}
    `),
  );
});

test('includes support for exporting default async functions', t => {
  t.snapshot(
    getAst(`
    export default async function f() {}
    `),
  );
});

test('includes no-line-terminator requirement for async functions', t => {
  t.snapshot(
    getAst(`
    async 
    function f() {}
    `),
  );
});

test('includes no-line-terminator requirement for async function expressions', t => {
  t.snapshot(
    getAst(`
    let f = async 
    function f() {}
    `),
  );
});

test('includes support for async arrow functions', t => {
  t.snapshot(
    getAst(`
    let f = async () => {}
    `),
  );
});

test('includes support for async method definitions', t => {
  t.snapshot(
    getAst(`
    class C {
      async f() {}
    }
    `),
  );
});

test('handles properties named async', t => {
  t.snapshot(
    getAst(`
    let o = {
      async: true
    }
    `),
  );
});

test('includes support for await', t => {
  t.snapshot(
    getAst(`
    async function f () {
      await g();
    }
    `),
  );
});
