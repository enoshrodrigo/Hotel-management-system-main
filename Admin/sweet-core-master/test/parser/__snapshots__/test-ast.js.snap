// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does include the use strict directive in the AST 1`] = `
Module {
  "directives": Array [
    Object {
      "rawValue": "use strict",
      "type": "Directive",
    },
  ],
  "items": Array [],
  "loc": null,
  "type": "Module",
}
`;

exports[`does not include the lang directive in the AST 1`] = `
Module {
  "directives": Array [],
  "items": Array [],
  "loc": null,
  "type": "Module",
}
`;

exports[`handles properties named async 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    VariableDeclarationStatement {
      "declaration": VariableDeclaration {
        "declarators": Array [
          VariableDeclarator {
            "binding": BindingIdentifier {
              "loc": null,
              "name": "o_4",
              "type": "BindingIdentifier",
            },
            "init": ObjectExpression {
              "loc": null,
              "properties": Array [
                DataProperty {
                  "expression": LiteralBooleanExpression {
                    "loc": null,
                    "type": "LiteralBooleanExpression",
                    "value": true,
                  },
                  "loc": null,
                  "name": StaticPropertyName {
                    "loc": null,
                    "type": "StaticPropertyName",
                    "value": "async",
                  },
                  "type": "DataProperty",
                },
              ],
              "type": "ObjectExpression",
            },
            "loc": null,
            "type": "VariableDeclarator",
          },
        ],
        "kind": "let",
        "loc": null,
        "type": "VariableDeclaration",
      },
      "loc": null,
      "type": "VariableDeclarationStatement",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes export declaration in AST 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    VariableDeclarationStatement {
      "declaration": VariableDeclaration {
        "declarators": Array [
          VariableDeclarator {
            "binding": BindingIdentifier {
              "loc": null,
              "name": "x_0",
              "type": "BindingIdentifier",
            },
            "init": LiteralNumericExpression {
              "loc": null,
              "type": "LiteralNumericExpression",
              "value": 1,
            },
            "loc": null,
            "type": "VariableDeclarator",
          },
        ],
        "kind": "var",
        "loc": null,
        "type": "VariableDeclaration",
      },
      "loc": null,
      "type": "VariableDeclarationStatement",
    },
    ExportLocals {
      "loc": null,
      "moduleSpecifier": null,
      "namedExports": Array [
        ExportLocalSpecifier {
          "exportedName": "x",
          "loc": null,
          "name": IdentifierExpression {
            "loc": null,
            "name": "x_0",
            "type": "IdentifierExpression",
          },
          "type": "ExportLocalSpecifier",
        },
      ],
      "type": "ExportLocals",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes export in AST 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    ExportLocals {
      "loc": null,
      "moduleSpecifier": null,
      "namedExports": Array [
        ExportLocalSpecifier {
          "exportedName": null,
          "loc": null,
          "name": IdentifierExpression {
            "loc": null,
            "name": "b",
            "type": "IdentifierExpression",
          },
          "type": "ExportLocalSpecifier",
        },
      ],
      "type": "ExportLocals",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes export with renaming in AST 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    ExportLocals {
      "loc": null,
      "moduleSpecifier": null,
      "namedExports": Array [
        ExportLocalSpecifier {
          "exportedName": "c",
          "loc": null,
          "name": IdentifierExpression {
            "loc": null,
            "name": "b",
            "type": "IdentifierExpression",
          },
          "type": "ExportLocalSpecifier",
        },
      ],
      "type": "ExportLocals",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes no-line-terminator requirement for async function expressions 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    VariableDeclarationStatement {
      "declaration": VariableDeclaration {
        "declarators": Array [
          VariableDeclarator {
            "binding": BindingIdentifier {
              "loc": null,
              "name": "f_2",
              "type": "BindingIdentifier",
            },
            "init": IdentifierExpression {
              "loc": null,
              "name": "async",
              "type": "IdentifierExpression",
            },
            "loc": null,
            "type": "VariableDeclarator",
          },
        ],
        "kind": "let",
        "loc": null,
        "type": "VariableDeclaration",
      },
      "loc": null,
      "type": "VariableDeclarationStatement",
    },
    FunctionDeclaration {
      "body": FunctionBody {
        "directives": Array [],
        "loc": null,
        "statements": Array [],
        "type": "FunctionBody",
      },
      "isAsync": undefined,
      "isGenerator": false,
      "loc": null,
      "name": BindingIdentifier {
        "loc": null,
        "name": "f_2",
        "type": "BindingIdentifier",
      },
      "params": FormalParameters {
        "items": Array [],
        "loc": null,
        "rest": null,
        "type": "FormalParameters",
      },
      "type": "FunctionDeclaration",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes no-line-terminator requirement for async functions 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    ExpressionStatement {
      "expression": IdentifierExpression {
        "loc": null,
        "name": "async",
        "type": "IdentifierExpression",
      },
      "loc": null,
      "type": "ExpressionStatement",
    },
    FunctionDeclaration {
      "body": FunctionBody {
        "directives": Array [],
        "loc": null,
        "statements": Array [],
        "type": "FunctionBody",
      },
      "isAsync": undefined,
      "isGenerator": false,
      "loc": null,
      "name": BindingIdentifier {
        "loc": null,
        "name": "f",
        "type": "BindingIdentifier",
      },
      "params": FormalParameters {
        "items": Array [],
        "loc": null,
        "rest": null,
        "type": "FormalParameters",
      },
      "type": "FunctionDeclaration",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes support for async arrow functions 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    VariableDeclarationStatement {
      "declaration": VariableDeclaration {
        "declarators": Array [
          VariableDeclarator {
            "binding": BindingIdentifier {
              "loc": null,
              "name": "f_3",
              "type": "BindingIdentifier",
            },
            "init": ArrowExpression {
              "body": FunctionBody {
                "directives": Array [],
                "loc": null,
                "statements": Array [],
                "type": "FunctionBody",
              },
              "isAsync": true,
              "loc": null,
              "params": FormalParameters {
                "items": Array [],
                "loc": null,
                "rest": null,
                "type": "FormalParameters",
              },
              "type": "ArrowExpression",
            },
            "loc": null,
            "type": "VariableDeclarator",
          },
        ],
        "kind": "let",
        "loc": null,
        "type": "VariableDeclaration",
      },
      "loc": null,
      "type": "VariableDeclarationStatement",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes support for async function declarations 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    FunctionDeclaration {
      "body": FunctionBody {
        "directives": Array [],
        "loc": null,
        "statements": Array [],
        "type": "FunctionBody",
      },
      "isAsync": true,
      "isGenerator": false,
      "loc": null,
      "name": BindingIdentifier {
        "loc": null,
        "name": "f",
        "type": "BindingIdentifier",
      },
      "params": FormalParameters {
        "items": Array [],
        "loc": null,
        "rest": null,
        "type": "FormalParameters",
      },
      "type": "FunctionDeclaration",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes support for async function expressions 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    VariableDeclarationStatement {
      "declaration": VariableDeclaration {
        "declarators": Array [
          VariableDeclarator {
            "binding": BindingIdentifier {
              "loc": null,
              "name": "f_1",
              "type": "BindingIdentifier",
            },
            "init": FunctionExpression {
              "body": FunctionBody {
                "directives": Array [],
                "loc": null,
                "statements": Array [],
                "type": "FunctionBody",
              },
              "isAsync": true,
              "isGenerator": false,
              "loc": null,
              "name": BindingIdentifier {
                "loc": null,
                "name": "f_1",
                "type": "BindingIdentifier",
              },
              "params": FormalParameters {
                "items": Array [],
                "loc": null,
                "rest": null,
                "type": "FormalParameters",
              },
              "type": "FunctionExpression",
            },
            "loc": null,
            "type": "VariableDeclarator",
          },
        ],
        "kind": "let",
        "loc": null,
        "type": "VariableDeclaration",
      },
      "loc": null,
      "type": "VariableDeclarationStatement",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes support for async functions 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    FunctionDeclaration {
      "body": FunctionBody {
        "directives": Array [],
        "loc": null,
        "statements": Array [],
        "type": "FunctionBody",
      },
      "isAsync": true,
      "isGenerator": false,
      "loc": null,
      "name": BindingIdentifier {
        "loc": null,
        "name": "f",
        "type": "BindingIdentifier",
      },
      "params": FormalParameters {
        "items": Array [],
        "loc": null,
        "rest": null,
        "type": "FormalParameters",
      },
      "type": "FunctionDeclaration",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes support for async method definitions 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    ClassDeclaration {
      "elements": Array [
        ClassElement {
          "isStatic": false,
          "loc": null,
          "method": Method {
            "body": FunctionBody {
              "directives": Array [],
              "loc": null,
              "statements": Array [],
              "type": "FunctionBody",
            },
            "isAsync": true,
            "isGenerator": false,
            "loc": null,
            "name": StaticPropertyName {
              "loc": null,
              "type": "StaticPropertyName",
              "value": "f",
            },
            "params": FormalParameters {
              "items": Array [],
              "loc": null,
              "rest": null,
              "type": "FormalParameters",
            },
            "type": "Method",
          },
          "type": "ClassElement",
        },
      ],
      "loc": null,
      "name": BindingIdentifier {
        "loc": null,
        "name": "C",
        "type": "BindingIdentifier",
      },
      "super": null,
      "type": "ClassDeclaration",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes support for await 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    FunctionDeclaration {
      "body": FunctionBody {
        "directives": Array [],
        "loc": null,
        "statements": Array [
          ExpressionStatement {
            "expression": AwaitExpression {
              "expression": CallExpression {
                "arguments": Array [],
                "callee": IdentifierExpression {
                  "loc": null,
                  "name": "g",
                  "type": "IdentifierExpression",
                },
                "loc": null,
                "type": "CallExpression",
              },
              "loc": null,
              "type": "AwaitExpression",
            },
            "loc": null,
            "type": "ExpressionStatement",
          },
        ],
        "type": "FunctionBody",
      },
      "isAsync": true,
      "isGenerator": false,
      "loc": null,
      "name": BindingIdentifier {
        "loc": null,
        "name": "f",
        "type": "BindingIdentifier",
      },
      "params": FormalParameters {
        "items": Array [],
        "loc": null,
        "rest": null,
        "type": "FormalParameters",
      },
      "type": "FunctionDeclaration",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes support for exporting async functions 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    FunctionDeclaration {
      "body": FunctionBody {
        "directives": Array [],
        "loc": null,
        "statements": Array [],
        "type": "FunctionBody",
      },
      "isAsync": true,
      "isGenerator": false,
      "loc": null,
      "name": BindingIdentifier {
        "loc": null,
        "name": "f",
        "type": "BindingIdentifier",
      },
      "params": FormalParameters {
        "items": Array [],
        "loc": null,
        "rest": null,
        "type": "FormalParameters",
      },
      "type": "FunctionDeclaration",
    },
    ExportLocals {
      "loc": null,
      "moduleSpecifier": null,
      "namedExports": Array [
        ExportLocalSpecifier {
          "exportedName": "f",
          "loc": null,
          "name": IdentifierExpression {
            "loc": null,
            "name": "f",
            "type": "IdentifierExpression",
          },
          "type": "ExportLocalSpecifier",
        },
      ],
      "type": "ExportLocals",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;

exports[`includes support for exporting default async functions 1`] = `
Module {
  "directives": Array [],
  "items": Array [
    ExportDefault {
      "body": FunctionExpression {
        "body": FunctionBody {
          "directives": Array [],
          "loc": null,
          "statements": Array [],
          "type": "FunctionBody",
        },
        "isAsync": true,
        "isGenerator": false,
        "loc": null,
        "name": BindingIdentifier {
          "loc": null,
          "name": "f",
          "type": "BindingIdentifier",
        },
        "params": FormalParameters {
          "items": Array [],
          "loc": null,
          "rest": null,
          "type": "FormalParameters",
        },
        "type": "FunctionExpression",
      },
      "loc": null,
      "type": "ExportDefault",
    },
  ],
  "loc": null,
  "type": "Module",
}
`;
